src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -147,6 +177,37 @@ export default class MutationMapperStore
         }
     }, undefined);
 
+    @computed get transcriptsWithAnnotations(): string[] {
```

Comment:
why not make this and transcriptsWithProteinLength mobxpromises?

Reply:
@adamabeshouse 	i guess im a bit confused about when to use computed over a promise. Could you clarify?

Reply:
the advantages to using a promise are that you can await it, you can check its status when using it. so you could show a loading indicator for the dropdown until these results were loaded. but if thats not an issue, then @computed should be fine

Reply:
i see thanks!